string var_name
string var_value
string var_type

int8 INT = 1
int8 FLOAT = 2
int8 STRING = 3
int8 BOOL = 4
int8 COORDINATE = 5


